paper.md

```diff
@@ -314,6 +324,7 @@ A live demo of more sophisticated Empirical prefabricated widgets, presented alo
 
 Empirical remains under active development.
 Current priorities include assembling compound, higher-level web widgets for common tasks, developing a comprehensive framework for digital evolution experiments, and adding more step-by-step tutorials to our documentation.
+Notably, preparations for a major v1.0 release integrating extensive feature development associated with the MABE2 project are nearing completion.
```

Comment:
This was the sentence that made me wonder if there were software development goals associated with this submission - maybe we should cut it in case reviewers feel the same way (and also because the goal of this is to be a long-term archival paper and this sentence will hopefully be out of date fairly soon!)

Reply:
Hmm, my rationale was less to point to software development goals and more to explain why the main branch has been relatively quiet lately (this concern being related to our discussion on the slack about putting the joss paper out now/near future). Any ideas for how to rephrase?

Reply:
I wonder if that's maybe a thing to mention to the editor during submission rather than putting it in the paper?

Reply:
That’s a really good point — I think that’s the best idea.